partiql-eval/src/lib.rs

```diff
@@ -2435,301 +2435,4 @@ mod tests {
             assert_eq!(Value::Bag(Box::new(expected)), res);
         }
     }
-
-    mod graph {
```

Comment:
I assume this is basically the same as what's now in `partiql-eval/src/eval/expr/graph_match.rs`?

Reply:
yes